### ISSUE TITLE
what are the differences among the 3 split models?

### ISSUE THREAD

**Issue by jiandan42** [feichtenhofer/twostreamfusion#14]
There are 3 models for each kind of pretrained models, for example, ucf101-img-vgg16-split1.mat, ucf101-img-vgg16-split2.mat and ucf101-img-vgg16-split3.mat. I find that you set nSplit=1 in the cnn_ucf101_fusion.m. What do the other two models mean and what are the differences among these three model? Thank you for your help.

**Comment by abursuc** [comment]
Hi,
The splits correspond to the ones defined by the authors of the UCF 101 dataset. Basically the dataset is split into train and test data three times. The dataset is not so big and this is just a manner to try to see how the models trained on it generalize to various types of the data. You should find out more about it on the UCF 101 web page http://crcv.ucf.edu/data/UCF101.php

**Comment by rocksyne** [comment]
@abursuc I have one quick question please. As I have read in a number of literatures, the datasets are usually split in 3.

- Training  set

- Validation  set and

- Test  set.

In the case of the train-test split of UCF101, do we split the Training set into two (Train set and Validation set)? I will be glad if you could give me some guidance on this please.
